nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -100,6 +100,7 @@ def do_one_task(self, client):
                     if client_runner.run_abort_signal.triggered:
                         stop_run = True
                         self.logger.info("End the Simulator run.")
+                        client_runner.end_run_events_sequence()
```

Comment:
Actually, this is not totally correct. Call the client_runner.end_run_events_sequence() here only means the current client finishes its running turn. The simulator running thread will pass the "running turn" to the next client to run. The end_run_events_sequence() will fire the END_RUN event, which the client may come back to run again.

Instead, the START_RUN and END_RUN event needs to be handled in simulator_runner's run_client_thread().